### PR TITLE
fix(upload): reduced EncryptionHeaderSize with 256

### PR DESCRIPTION
**Pull request by cnlangzi**
+ zbox: https://github.com/0chain/zboxcli/pull/173
  + gosdk: https://github.com/0chain/gosdk/pull/374
  + blobber: https://github.com/0chain/blobber/pull/538
  + system_test: https://github.com/0chain/system_test/pull/195

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/reduce_encryption_header_size  |
| gosdk  | fix/reduce_encryption_header_size  |
| zboxcli  | fix/reduce_encryption_header_size  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | fix/reduce_encryption_header_size  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by stewartie4 — APPROVED** [review]

File: tests/cli_tests/zboxcli_upload_test.go [hunk 1]
```diff
@@ -717,7 +717,7 @@ func TestUpload(t *testing.T) {
 			"allocation": allocationID,
 			"remotepath": "/" + filepath.Base(fileName),
 			"localpath":  fileName,
-			"chunksize":  1024,
+			"chunksize":  255,
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_upload_test.go:720]: The error message should now be
Upload failed. chunk: chunk size is too small. it must greater than **256** if file is uploaded with encryption
right?

Comment (cnlangzi) [reply tests/cli_tests/zboxcli_upload_test.go:720]: 256+16 = EncryptionHeaderSize + EncryptedDataPaddingSize

**Review by stewartie4 — APPROVED** [review]
LGTM Thanks Dayi

**Pull request merged**